docusaurus/docs/installation.md

```diff
@@ -114,8 +114,34 @@ Reassure CLI will automatically try to detect your source code branch name and c
 ```sh
 yarn reassure --branch [branch name] --commit-hash [commit hash] 
 ```
+## CI setup
+To make setting up the CI integration and all prerequisites more convenient, we have prepared a CLI command which will generate all necessary templates for you to get started with. 
 
-## Write performance testing script
+Simply run:
+```bash
+yarn reassure init --logLevel <verbose|default|silent>
+```
+This will generate the following file structure
+```
+├── <ROOT>
+│   ├── .reassure
+│   │   └──  reassure-tests.sh
+│   ├── dangerfile.ts (conditional)
+│   ├── dangerfile.reassure.ts (conditional)
+│   └── .gitignore (conditional)
+```
+
+#### `--logLevel` (optional)
+This option allows you to control the level of logs. By default, you will only see errors and warnings if those arise, however, if you would like to have a bit more insight into the process you can run it with `--logLevel verbose`. Alternatively, you might want to skip the logs in their entirety. In that case, you can run the command with `--logLevel silent`. 
+#### `dangerfile.ts and dangerfile.reassure.ts`
```

Comment:
Maybe we can always generate only `dangerfile.reassure.ts` instead of generating `dangerfile.ts`. Wdyt @Xiltyn ?

Reply:
we could absolutely do that, this removes the issue of potential merging, which is way trickier. On the other hand, is there a point to remove and optimisation which makes it easier for ppl that don't have any dangerfile yet?

wdyt?

Reply:
I made a change in the docs so that it reflects both of these scenarios better. End of the day I think it's better to make easier for ppl who aren't well versed in danger.js, but I'm happy to hear your opinion further :)

Reply:
The current solution is good. No need to tweak it.